ide-support/revsaveasandroidstandalone.livecodescript

```diff
@@ -521,11 +521,11 @@ private command revSaveAsMobileStandaloneMain pStack, pApkFile, pTarget, pSettin
       replace "${PUSH_PERMISSIONS}" with tPushPermissions in tManifest
       replace "${PUSH_RECEIVER}" with tPushReceiver in tManifest
       
-      -- SN-2014-02-07: Add the email attachments provider in the manifest
+      // Add the file provider to the manifest
       local tProvider
       put "<provider " & return into tProvider
       put "      android:name=" & quote & pSettings["android,identifier"] & ".AppProvider" & quote & return after tProvider
-      put "      android:authorities=" & quote & pSettings["android,identifier"] & ".attachmentprovider" & quote & return after tProvider
+      put "      android:authorities=" & quote & pSettings["android,identifier"] & ".fileprovider" & quote & return after tProvider
       put "      android:exported=" & quote & "true" & quote & return after tProvider
```

Comment:
No, exported=true make the content accessible to all apps instead of just the app launched to handle the intent. I don't think AndroidManifest.xml is used for anything other than launching test apps from the command line.

Reply:
Ok makes sense now. Thanks @livecodeian :)

Reply:
Sorry Panos, looking at this again I think you're right. We want to limit access to the content uri to the app we launch through the intent so this should be false. Will need to test first to make sure it doesn't break email attachments & camera capture first :)